### DISCUSSION TITLE
S2 - Style, how to add pseudo

### DISCUSSION THREAD

**Discussion by ladifire** [adobe/react-spectrum#7544] [General]
Hi @devongovett , I'm not sure how to add `pseudo` with S2-style macro. I've tried this but not luck:
<img width="380" alt="image" src="https://github.com/user-attachments/assets/9858e827-05e3-456c-92f2-3d54e824647b" />
Could you please advice? thanks!

**Comment by devongovett** [comment]
This is intentionally not supported. We recommend using a real DOM node as a child instead.
